.appveyor.yml

```
@@ -19,7 +19,8 @@ clone_script:
       && git submodule update --init --recursive
 
 environment:
-  APPVEYOR_SAVE_CACHE_ON_ERROR : true
+  APPVEYOR_SAVE_CACHE_ON_ERROR : false
```

yamllint

[error] too many spaces before colon

(colons)